jbt-it/neomdb
# #83 client edit link at departmentoverview

- Added department dialog component
- Added update department route
- Added api call to update departments

**How to test**:
- Look at code and check understandability and correctness
- Navigate to '/ressorts' with a user who is a 'Ressortleiter'
- Change links of your department and check if these changes are persisted
